Recast/SeriesViewController.swift

```diff
@@ -1,20 +1,80 @@
 //
-//  ViewController.swift
+//  SeriesViewController.swift
 //  Recast
 //
-//  Created by Drew Dunne on 9/12/18.
+//  Created by Jack Thompson on 9/15/18.
 //  Copyright © 2018 Cornell AppDev. All rights reserved.
 //
 
 import UIKit
+import SnapKit
 
 class SeriesViewController: UIViewController {
-
+    
+    /// MARK: Variables
+    var headerView: SeriesHeaderView!
+    var episodeTableView: UITableView!
+    
+    /// MARK: Constants
+    let reuseIdentifer = "episodeCell"
+    let headerHeight:CGFloat = 200
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
-        view.backgroundColor = .blue
+        view.backgroundColor = .white
+        
+        headerView = SeriesHeaderView(frame: .zero)
+        
+        episodeTableView = UITableView()
+        episodeTableView.delegate = self
+        episodeTableView.dataSource = self
+        episodeTableView.rowHeight = UITableViewAutomaticDimension
+        episodeTableView.tableHeaderView = headerView
+        
+        view.addSubview(episodeTableView)
+        
+        
+        layoutSubviews()
+    }
+    
+    func layoutSubviews() {
+        headerView.snp.makeConstraints { (make) in
+            make.edges.equalToSuperview()
+            make.width.equalTo(view.safeAreaLayoutGuide)
+        }
+        episodeTableView.snp.makeConstraints { (make) in
+            make.edges.equalToSuperview()
+        }
+        
+        episodeTableView.tableHeaderView?.layoutIfNeeded()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
     }
 
 }
 
+///MARK: episodeTableView Delegate & DataSource
```

Comment:
Can you separate these out into delegate and data source extensions? Use `// MARK - `